src/NzbDrone.Core.Test/MediaFiles/UpdateEpisodeFileServiceTests/ChangeFileDateFixture.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Reflection;
+using Moq;
+using NUnit.Framework;
+using NzbDrone.Common.Disk;
+using NzbDrone.Common.EnvironmentInfo;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.MediaFiles;
+using NzbDrone.Core.Test.Framework;
+
+namespace NzbDrone.Core.Test.MediaFiles.UpdateEpisodeFileServiceTests
+{
+    [TestFixture]
+    public class ChangeFileDateFixture : CoreTest<UpdateEpisodeFileService>
+    {
+        private const string FilePath = "dummy.file";
+        private DateTime _lastWrite;
+
+        private bool InvokePrivate(string path, DateTime airDate)
```

Comment:
Don't call private methods, the public interface should be used for all tests.

To check whether the date is updated you can use `Verify` on the mock.